content/strategy-goals/strategy/code-graph/code-security-tiger-team/index.md

```diff
@@ -2,38 +2,26 @@
 
 The purpose of the Code Security [tiger team](https://en.wikipedia.org/wiki/Tiger_team) is to iterate quickly with customers over new user flows addressing the [Code security use case](../../use-cases/code-security.md) over FY23Q2.
 
-Our strategy is work in progress, but we will focus heavily on running customer discovery in May.
+#### Quicklink
 
-#### What success means
+- [Team page](../../../../departments/engineering/dev/code-graph/code-security-tiger-team/index.md)
+- [OKR](https://docs.google.com/document/d/1wh8enIZs8mDEoBs_HiPP4v9SIF2gFuhtMFAhbROe2EI/edit#heading=h.tsga00gx9kb8) (private)
+- [Weekly updates](https://github.com/sourcegraph/code-graph/issues/21)
 
-We are in customer discovery phase, and will define a success metric later on. Our main business goals are:
+Following early customer discovery, we are making a bet on software supply chain. We are iterating rapidly, so what follows may change as we learn. You can also read more in [PD 35 WIP - Dependency Graph for code security](https://docs.google.com/document/d/1Bq2SAx-FXxzW0YFbkIYoPS6bvy-LvlvZMyOTNUI1NV0/edit#heading=h.z5ozjpfzh2yl).
 
-- Close deals faster (with focused, quicker new business lands), and a reduced average sales cycle
-- Attach to a compelling event
-- Fill a meaningful gap in the current security tool landscape
-- Create incremental and meaningful value for our customers
-- Provide a new buyer to land with in target accounts
+### Vision
 
-#### Plan
+Security scanners have a low signal to noise ratio. This means that security and development teams have to spend more and more time triaging and fixing vulnerabilities. This is made more difficult because there is a gap bewteen security and development goals and workflows: security teams want to ensure compliance and reduce risks, while development teams have to make trade-offs between updating dependencies and delivering on other priorities.
```

Comment:
```suggestion
Security scanners have a low signal to noise ratio. This means that security and development teams have to spend more and more time triaging and fixing vulnerabilities. This is made more difficult because there is a gap between security and development goals and workflows: security teams want to ensure compliance and reduce risks, while development teams have to make trade-offs between updating dependencies and delivering on other priorities.
```